<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room</class>
 <widget class="QMainWindow" name="room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{border-image: url(:/image/login/preview.jpg);}</string>
   </property>
   <widget class="QWidget" name="led1" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/closeled_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="led2" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>220</y>
      <width>120</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/closeled_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="feng" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>400</y>
      <width>120</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/closefan_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="door" native="true">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>120</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/closedoor_ico.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="led1button">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ledbutton2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="doorbutton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>720</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fengbutton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
